src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -318,19 +318,30 @@ static int GetNdkApiLevel(string androidNdkPath, string androidApiLevel, Android
 					string seqpointsFile = Path.Combine(outdir, string.Format ("{0}.msym",
 						Path.GetFileName (assembly.ItemSpec)));
 
-					string aotOptions = string.Format (
-						"{0}--aot={9}{8}{1}outfile={2},asmwriter,mtriple={3},tool-prefix={4},ld-flags={5},llvm-path={6},temp-path={7}",
-						EnableLLVM ? "--llvm " : string.Empty,
-						AotMode != AotMode.Normal ? string.Format("{0},", AotMode.ToString().ToLowerInvariant()) : string.Empty,
-						QuoteFileName (outputFile),
-						mtriple,
-						QuoteFileName (toolPrefix),
-						ldFlags,
-						QuoteFileName (SdkBinDirectory),
-						QuoteFileName (outdir),
-						sequencePointsMode == SequencePointsMode.Offline ? string.Format("msym-dir={0},", QuoteFileName(outdir)) : string.Empty,
-						AotAdditionalArguments != string.Empty ? string.Format ("{0},", AotAdditionalArguments) : string.Empty
-					);
+					List<string> aotOptions = new List<string> ();
+
+					if (!string.IsNullOrEmpty (AotAdditionalArguments))
+						aotOptions.Add (AotAdditionalArguments);
+					if (sequencePointsMode == SequencePointsMode.Offline)
+						aotOptions.Add ("msym-dir=" + QuoteFileName(outdir));
+					if (AotMode != AotMode.Normal)
+						aotOptions.Add (AotMode.ToString().ToLowerInvariant());
+
+					aotOptions.Add ("outfile="     + QuoteFileName (outputFile));
+					aotOptions.Add ("asmwriter");
+					aotOptions.Add ("mtriple="     + mtriple);
+					aotOptions.Add ("tool-prefix=" + QuoteFileName (toolPrefix));
+					aotOptions.Add ("ld-flags="    + ldFlags);
+					aotOptions.Add ("llvm-path="   + QuoteFileName (SdkBinDirectory));
+					aotOptions.Add ("temp-path="   + QuoteFileName (outdir));
+
+					if (Environment.GetEnvironmentVariable ("MONO_AOT_LOG_SKIPPED") != null)
```

Comment:
Should this *really* be via environment variables, or should we add an additional Property to this task for these, e.g.:

    public string ExtraAotOptions {get; set;}

Otherwise, the only way to actually set these things will be by invoking on the command-line:

    $ MONO_AOT_LOG_INSTANCES=1 msbuild /t:SignAndroidPackage Project.csproj ...